doc/source/user/index.rst

```diff
@@ -0,0 +1,26 @@
+:orphan:
+
+.. _user:
+
+################
+NumPy User Guide
+################
+
+This guide is intended as an introductory overview of NumPy and
+explains how to install and make use of the most important features of
+NumPy. For detailed reference documentation of the functions and
+classes contained in the package, see the :ref:`reference`.
+
+.. toctree::
+   :maxdepth: 1
+
+   setting-up
+   quickstart
+   absolute_beginners
+   basics
+   misc
+   numpy-for-matlab-users
+   building
+   c-info
+   tutorials_index
```

Comment:
this document should be tutorial-svd?  because tutorials_index is almost the same with index.rst.

Reply:
So, maybe this should be remapped.  `index.rst` should contain everything under "For users" in this page: https://numpy.org/devdocs/

`tutorials_index.rst` should be just the tutorials (although some of them might show up twice because the quickstart tutorial and the absolute beginners tutorial show up at the main index page).

Reply:
> this document should be tutorial-svd? because tutorials_index is almost the same with index.rst.

The reason for including the index rather than the tutorials individually is that there will be more tutorials in the future, and this should capture them automatically. Sphinx is smart enough to non replicate redundant entries in the toc, so the quickstart, absolute beginner, etc. tutorials are not duplicated (also, see below about user guide layout).

> So, maybe this should be remapped. index.rst should contain everything under "For users" in this page: https://numpy.org/devdocs/

In the current layout (with the *_index files included, but the original toc entries not removed) each entry gets it's own chapter in the user guide. If we remove things like `basics`, `c-info`, `building` etc. from the toc here in favor of the `tutorials_index`, then these entries become sections in a larger chapter called "tutorials". In other words, the only thing affected on the `latex` side of things is the layout of the user guide: it's a matter of preference whether we have individual chapters for "miscellaneous", "using the c-api", etc., or one big chapter with these as sections.

Since the content is the same, I don't mind either way - if you prefer the "big chapter" approach over the current layout, LMK and I will modify it!

Reply:
On second thought, this might be a nice place for the `.. only` directive: we can use one toctree to explicitly control chapter layout for the latex-version of the user guide, and a different one for the html output that matches the layout of the "For users" section in the html docs.

Reply:
I am pretty sure you should avoid the rabbit-hole of `.. only` since I do not think it works very well.

Reply:
I think this is fine as is, if needs change in the near future we can always open a new issue. Thanks, @rossbar :)